### ISSUE TITLE
Enable ReceiverTests again, when Mockito releases a newer >5.5.0 version

### ISSUE THREAD

**Issue by jkosternl** [ibissource/iaf#5547]
Currently, testPollGuardStartTimeout and testPollGuardStopTimeout are put on disabled, because they fail. The failure is caused by mocks which take > 5 seconds to initiate with Java 11. This only happens when running all unit tests inside the Core project. 

Line `Mockito.clearAllCaches();` did fix it on my local laptop, but in Jenkins, it still fails. 
When Mockito releases a newer version, or we're moved to Java 17, retry enabling these tests, to see if they are working again. 

Useful stacktrace:
```
10:23:54,311 DEBUG TimeoutGuard$Killer.run():58[] Execution stack for thread [starting Adapter ReceiverTestAdapterName] executing task [starting receiver [receiver]]:
    at java.base@11.0.20.1/java.lang.invoke.MethodHandleNatives.init(Native Method)
    at java.base@11.0.20.1/java.lang.invoke.MemberName.<init>(MemberName.java:668)
    at java.base@11.0.20.1/java.lang.invoke.MethodHandles$Lookup.unreflectConstructor(MethodHandles.java:1826)
    at app//org.mockito.internal.util.reflection.InstrumentationMemberAccessor.newInstance(InstrumentationMemberAccessor.java:185)
    at app//org.mockito.internal.util.reflection.ModuleMemberAccessor.newInstance(ModuleMemberAccessor.java:49)
    at app//org.mockito.internal.creation.bytebuddy.InlineDelegateByteBuddyMockMaker.newInstance(InlineDelegateByteBuddyMockMaker.java:631)
    at app//org.mockito.internal.creation.bytebuddy.InlineDelegateByteBuddyMockMaker.doCreateMock(InlineDelegateByteBuddyMockMaker.java:369)
    at app//org.mockito.internal.creation.bytebuddy.InlineDelegateByteBuddyMockMaker.createMock(InlineDelegateByteBuddyMockMaker.java:334)
    at app//org.mockito.internal.creation.bytebuddy.InlineByteBuddyMockMaker.createMock(InlineByteBuddyMockMaker.java:56)
    at app//org.mockito.internal.util.MockUtil.createMock(MockUtil.java:99)
    at app//org.mockito.internal.MockitoCore.mock(MockitoCore.java:88)
    at app//org.mockito.Mockito.mock(Mockito.java:2101)
    at app//org.mockito.Mockito.mock(Mockito.java:2016)
    at app//nl.nn.adapterframework.receivers.SlowListenerWithPollGuard.open(SlowListenerWithPollGuard.java:38)
    at app//nl.nn.adapterframework.receivers.Receiver.openAllResources(Receiver.java:435)
    at app//nl.nn.adapterframework.receivers.Receiver.startRunning(Receiver.java:784)
    at app//nl.nn.adapterframework.core.Adapter$1.run(Adapter.java:833)
    at java.base@11.0.20.1/java.lang.Thread.run(Thread.java:829)
```

**Comment by tnleeuw** [comment]
I'm beginning to think that these tests have timeouts creating mocks because the mocks are created not on the main thread but a background thread?

**Comment by tnleeuw** [comment]
I think these tests will be fixed with this PR: https://github.com/ibissource/iaf/pull/5610

**Comment by jkosternl** [comment]
> I think these tests will be fixed with this PR: #5610

Since the tests still run fine, with Java 11, you're right. I'll close this issue.

**Comment by tnleeuw** [comment]
> > I think these tests will be fixed with this PR: #5610
> 
> Since the tests still run fine, with Java 11, you're right. I'll close this issue.

Yes, Niels tested this yesterday before merging the Java11 changes to master.
